### PR TITLE
Icon list

### PR DESCRIPTION
**Pull request by una**
Basic Icon list appearing based on JSON data.

Looks like:
screen shot 2016-01-05 at 9 46 18 am

### CODE REVIEW

File: app/js/App.js [hunk 1]
```diff
@@ -1,78 +1,45 @@
...
-
   render() {
     return (
+
       <div>
 
         <Header />
 
-        {this.renderChildren()}
+          <IconList icons={this.state.icons} />
```
Comment (rmurphey) [inline app/js/App.js:34]: I don't know too much about the app -- are you envisioning that the icons in the list will change? If so, then using state is great; if not, you'd want to use props.

File: app/js/data/iconListInfo.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// JSON Object of all of the icons and their tags
+
+const icons = {
```
Comment (rmurphey) [inline app/js/data/iconListInfo.js]: More idiomatic here would just be to export the object itself, and skip giving it a name.

File: app/js/data/iconListInfo.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+// JSON Object of all of the icons and their tags
```
Comment (rmurphey) [inline app/js/data/iconListInfo.js:1]: For future consideration: you can imagine in the future that this list might be loaded asynchronously from a database. You might consider having this module export a node-style async function:

``` js
export default function (err, cb) {
  let iconList = { 
    // ...
  };
  cb(iconList);
}
```

File: package.json [hunk 4]
```diff
@@ -31,6 +31,9 @@
     "debowerify": "^1.3.1",
     "del": "^2.2.0",
     "eslint-plugin-react": "^3.6.3",
+    "connect-history-api-fallback": "1.1.0",
+    "firebase": "2.3.1",
+    "history": "^1.17.0",
```
Comment (rmurphey) [inline package.json]: It seems like these additions aren't related to the PR ... to reduce confusion later, you might want to remove these from here if they're not needed.

**Comment by rmurphey** [comment]
The code looks great -- I'd love it if the PR description gave me some details on how to see it in action :)

File: app/js/App.js (hunk 1 above)
Comment (una) [inline app/js/App.js:34]: The overall icon list won't change but individual pages will have their own smaller lists attached. I guess what I should do is remove icons from state and pull them in within the IconList object where they're being used for now

**Pull request merged**